database/v2/schemas/public_profile.model.js

```diff
@@ -0,0 +1,41 @@
+module.exports = (sequelize, type) => {
+  const MODEL = sequelize.define(
+    'PublicProfiles',
+    {
+      id: {
+        type: type.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      twitterLink: {
+        type: type.TEXT,
+        allowNull: true,
+      },
+      linkedInLink: {
+        type: type.TEXT,
+        allowNull: true,
+      },
+      faceBookLink: {
+        type: type.TEXT,
+        allowNull: true,
```

Comment:
Replace `faceBookLink` to `facebookLink`. Yes, looks like a joke, but this is an official name)

Reply:
i don't think link is good naming for identifier, also i think we can avoid using Text type here. Make investigate about size of facebook,linkedin token

Reply:
Ok. I'll search for such information/

Reply:
So those fields are for storing URL to their accounts.
We decided to not store token